components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/AuthenticationValve.java

```diff
@@ -173,6 +180,10 @@ public void invoke(Request request, Response response) throws IOException, Servl
         } catch (PatternSyntaxException e) {
             log.debug("Invalid pattern syntax of the request: ", e);
             APIErrorResponseHandler.handleErrorResponse(null, response, HttpServletResponse.SC_BAD_REQUEST, null);
+        } catch (DCRMException e) {
+            log.debug("Error while getting DCR Configuration: ", e);
```

Comment:
shouldn't this has to be an error log?

Reply:
Fixed with [1a9357b](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/269/commits/1a9357bf64f76553525bb96ff2932181094deb39)